File: 2026-1/sesion-de-clase-03/dotnet/seguro-vehicular/Seguros/SeguroOro.cs
namespace Seguros;

public class SeguroOro : SeguroPlata
{
    public SeguroOro(Vehiculo vehiculo) : base(vehiculo)
    {
    }

    public override double CalcularCosto()
    {
        return base.CalcularCosto() + 300.00;
    }

    public override string Descripcion()
    {
        return base.Descripcion() + " + cobertura contra desastres naturales";
    }
}

File: 2026-1/bk/sesion-de-clase-04/java/seguro-vehicular/pe/edu/segurosoft/seguros/SeguroOro.java
package pe.edu.segurosoft.seguros;

import pe.edu.segurosoft.modelo.Vehiculo;

public class SeguroOro extends SeguroPlata {

    public SeguroOro(Vehiculo vehiculo) {
        super(vehiculo);
    }

    @Override
    public double CalcularCosto() {
        return super.CalcularCosto() + 300.00;
    }

    @Override
    public String Descripcion() {
        return super.Descripcion() + " + cobertura contra desastres naturales";
    }
}
